helm/ffc-demo-payment-service/templates/container-secret.yaml

```diff
@@ -1,19 +1,15 @@
 {{- include "ffc-helm-library.container-secret" (list . "ffc-demo-payment-service.container-secret") -}}
 {{- define "ffc-demo-payment-service.container-secret" -}}
 stringData:
-  POSTGRES_PASSWORD: {{ .Values.postgresService.postgresPassword | quote }}
-  POSTGRES_USERNAME: {{ .Values.postgresService.postgresUsername | quote }}
-  PAYMENT_QUEUE_USER: {{ .Values.container.paymentQueueUser | quote }}
-  PAYMENT_QUEUE_PASSWORD: {{ .Values.container.paymentQueuePassword | quote }}
-  SCHEDULE_QUEUE_USER: {{ .Values.container.scheduleQueueUser | quote }}
-  SCHEDULE_QUEUE_PASSWORD: {{ .Values.container.scheduleQueuePassword | quote }}
+  APPINSIGHTS_INSTRUMENTATIONKEY: {{ quote .Values.appInsights.key }}
   {{- if .Values.b2c.url }}
-  B2C_CLIENT_SECRET: {{ .Values.b2c.clientSecret | quote }}
   B2C_CLIENT_ID: {{ .Values.b2c.clientId | quote }}
+  B2C_CLIENT_SECRET: {{ .Values.b2c.clientSecret | quote }}
   {{- end }}
   {{- if .Values.okta.domain }}
-  OKTA_CLIENT_ID: {{ .Values.okta.clientId | quote }}
   OKTA_AUTH_SERVER_ID: {{ .Values.okta.authorizationServerId | quote }}
+  OKTA_CLIENT_ID: {{ .Values.okta.clientId | quote }}
   {{- end }}
-  APPINSIGHTS_INSTRUMENTATIONKEY: {{ quote .Values.appInsights.key }}
+  POSTGRES_PASSWORD: {{ .Values.postgresService.postgresPassword | quote }}
```

Comment:
Are you also going to integrate the managed identity postgres changes into this PR too?

Reply:
I was going to do it as a separate PR for ease of review and keeping it focused.